### PR TITLE
FCN 번역

### PR DESCRIPTION
**Pull request by cepiloth**
FCN 번역

## 라이선스 동의
_변경해주시는 내용에 BSD 3항 라이선스가 적용됨을 동의해주셔야 합니다._<br />
_더 자세한 내용은 [기여하기 문서](https://github.com/PyTorchKorea/hub-kr/blob/master/CONTRIBUTING.md)를 참고해주세요._<br />
_동의하시면 아래 `[ ]`를 `[x]`로 만들어주세요._<br />

- [x] 기여하기 문서를 확인하였으며, 본 PR 내용에 BSD 3항 라이선스가 적용됨에 동의합니다.

## 관련 이슈 번호
_이 Pull Request와 관련있는 이슈 번호를 적어주세요._<br />
_이슈 또는 PR 번호 앞에 #을 붙이시면 제목을 바로 확인하실 수 있습니다. (예. #999 )_

- **이슈 번호**: #53 

## PR 종류
_이 PR에 해당되는 종류 앞의 `[ ]`을 `[x]`로 변경해주세요._<br />
- [ ] 오탈자를 수정하거나 번역을 개선하는 기여
- [x] 번역되지 않은 모델 소개를 번역하는 기여
- [ ] 공식 허브의 내용을 반영하는 기여
- [ ] 위 종류에 포함되지 않는 기여

## PR 설명
_이 PR로 무엇이 달라지는지 대략적으로 알려주세요._

### CODE REVIEW

File: pytorch_vision_fcn_resnet101.md [hunk 1]
```diff
@@ -25,26 +25,21 @@ model = torch.hub.load('pytorch/vision:v0.10.0', 'fcn_resnet50', pretrained=True
...
-and `std = [0.229, 0.224, 0.225]`.
+모든 사전 훈련된 모델은 동일한 방식으로 정규화된 입력 이미지, 즉 형태(N, 3, H, W)의 3채널 RGB 이미지의 미니배치는 N이 이미지 수이고, H와 W는 최소 224픽셀이 될 것으로 요구됩니다. 이미지를 [0, 1] 범위로 로드한 다음 mean = [0.485, 0.456, 0.406] 및 std = [0.229, 0.224, 0.225]를 사용하여 정규화해야 합니다.
 
-The model returns an `OrderedDict` with two Tensors that are of the same height and width as the input Tensor, but with 21 classes.
-`output['out']` contains the semantic masks, and `output['aux']` contains the auxillary loss values per-pixel. In inference mode, `output['aux']` is not useful.
-So, `output['out']` is of shape `(N, 21, H, W)`. More documentation can be found [here](https://pytorch.org/vision/stable/models.html#object-detection-instance-segmentation-and-person-keypoint-detection).
+모델은 입력 텐서와 높이와 너비는 같지만 클래스가 21개인 텐서를 가진 'Ordered Dict'를 반환합니다. output['out']에는 시멘틱 마스크가 포함되며 output['aux']에는 픽셀당 보조 손실 값이 포함됩니다. 추론 모드에서는 output['aux']이 유용하지 않습니다.그래서 output['out']의 크기는 (N, 21, H, W)이다. 추가 설명서는 [여기]에서 찾을 수 있습니다.(https://pytorch.org/vision/stable/models.html#object-detection-instance-segmentation-and-person-keypoint-detection).
```
Comment (HyunKyungHan) [inline pytorch_vision_fcn_resnet101.md:30]: 원문에 쓰인 대로 첫 번째 줄의 `Ordered Dict` -> `OrderedDict`로 띄어쓰기 없이 쓰는 게 좋을 것 같습니다.

**Review by HyunKyungHan — CHANGES_REQUESTED** [review]
몇 가지 사소한 수정 사항 제안드립니다! :)

File: pytorch_vision_fcn_resnet101.md [hunk 2]
```diff
@@ -25,26 +25,21 @@ model = torch.hub.load('pytorch/vision:v0.10.0', 'fcn_resnet50', pretrained=True
 model.eval()
 ```
 
-All pre-trained models expect input images normalized in the same way,
-i.e. mini-batches of 3-channel RGB images of shape `(N, 3, H, W)`, where `N` is the number of images, `H` and `W` are expected to be at least `224` pixels.
-The images have to be loaded in to a range of `[0, 1]` and then normalized using `mean = [0.485, 0.456, 0.406]`
-and `std = [0.229, 0.224, 0.225]`.
+모든 사전 훈련된 모델은 동일한 방식으로 정규화된 입력 이미지, 즉 형태(N, 3, H, W)의 3채널 RGB 이미지의 미니배치는 N이 이미지 수이고, H와 W는 최소 224픽셀이 될 것으로 요구됩니다. 이미지를 [0, 1] 범위로 로드한 다음 mean = [0.485, 0.456, 0.406] 및 std = [0.229, 0.224, 0.225]를 사용하여 정규화해야 합니다.
```
[the remark this answers is not in the document]
Comment (cepiloth) [reply pytorch_vision_fcn_resnet101.md:28]: 감사합니다. 제가볼때는 안보이는 것들이 리뷰를 통해서 알수있네요.^^

File: pytorch_vision_fcn_resnet101.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (cepiloth) [reply pytorch_vision_fcn_resnet101.md:30]: 좋은 리뷰 감사합니다.

Comment (cepiloth) [reply pytorch_vision_fcn_resnet101.md:30]: 좋은 리뷰 감사드립니다.

Comment (zhoonit) [inline pytorch_vision_fcn_resnet101.md:30]: 사소한 내용이지만, 향후 리뷰와 diff 를 위해 원문과 줄 수를 맞춰주시면 더 좋을것같습니다!

File: pytorch_vision_fcn_resnet101.md [hunk 3]
```diff
@@ -67,18 +62,15 @@ with torch.no_grad():
...
 
 ```python
-# create a color pallette, selecting a color for each class
+# 각 클래스에 대한 색상을 선택하여 색상 팔레트를 만듭니다.
 palette = torch.tensor([2 ** 25 - 1, 2 ** 15 - 1, 2 ** 21 - 1])
 colors = torch.as_tensor([i for i in range(21)])[:, None] * palette
 colors = (colors % 255).numpy().astype("uint8")
 
-# plot the semantic segmentation predictions of 21 classes in each color
+# 각 색상의 21개 클래스의 시멘틱 세그멘테이션 분할 예측을 그림으로 표시합니다.
```
Comment (zhoonit) [inline pytorch_vision_fcn_resnet101.md:73]: `시멘틱 세그멘테이션 분할 예측` 을 `시멘틱 세그멘테이션 예측` 정도로 번역하는 것은 어떨까요?

File: pytorch_vision_fcn_resnet101.md [hunk 4]
```diff
@@ -87,12 +79,11 @@ plt.imshow(r)
 # plt.show()
 ```
 
-### Model Description
+### 모델 설명
 
-FCN-ResNet is constructed by a Fully-Convolutional Network model, using a ResNet-50 or a ResNet-101 backbone.
-The pre-trained models have been trained on a subset of COCO train2017, on the 20 categories that are present in the Pascal VOC dataset.
+FCN-ResNet은 ResNet-50 또는 ResNet-101 백본을 사용하여 완전 컨볼루션 네트워크 모델로 구성됩니다. 사전 훈련된 모델은 Pascal VOC 데이터 세트에 존재하는 20개 범주에 대한 COCO 2017의 하위 집합에 대해 훈련 되었습니다.
```
Comment (zhoonit) [inline pytorch_vision_fcn_resnet101.md:84]: 완전 컨볼루션 네트워크 를 순화하여 컨볼루션만을 사용한 네트워크?? 정도로 번역하거나 원문 병행 표기는 어떨지 의견드립니다 :)

File: pytorch_vision_fcn_resnet101.md [hunk 5]
```diff
@@ -87,12 +79,11 @@ plt.imshow(r)
...
 
-Their accuracies of the pre-trained models evaluated on COCO val2017 dataset are listed below.
+COCO val 2017 데이터 세트에서 평가된 사전 훈련된 모델의 정확성은 아래에 나열되어 있습니다.
```
Comment (Taeyoung96) [inline pytorch_vision_fcn_resnet101.md:86]: [TRANSLATION_GUIDE.md](https://github.com/PyTorchKorea/tutorials-kr/blob/master/TRANSLATION_GUIDE.md)을 따르면 `데이터 세트`를 `데이터셋`이라고 바꾸는 것이 적절해 보입니다. 😄

**Review by Taeyoung96 — COMMENTED** [review]
번역하시느라 고생 많으셨습니다! 😃

File: pytorch_vision_fcn_resnet101.md (hunk 1 above)
Comment (cepiloth) [reply pytorch_vision_fcn_resnet101.md:30]: 리뷰감사드립니다.

File: pytorch_vision_fcn_resnet101.md (hunk 3 above)
Comment (cepiloth) [reply pytorch_vision_fcn_resnet101.md:73]: 참고하도록하겠습니다.

File: pytorch_vision_fcn_resnet101.md (hunk 5 above)
Comment (cepiloth) [reply pytorch_vision_fcn_resnet101.md:86]: 감사합니다. 참고하도록하겠습니다.

File: pytorch_vision_fcn_resnet101.md [hunk 6]
```diff
@@ -67,18 +62,15 @@ with torch.no_grad():
 output_predictions = output.argmax(0)
 ```
 
-The output here is of shape `(21, H, W)`, and at each location, there are unnormalized probabilities corresponding to the prediction of each class.
-To get the maximum prediction of each class, and then use it for a downstream task, you can do `output_predictions = output.argmax(0)`.
-
-Here's a small snippet that plots the predictions, with each color being assigned to each class (see the visualized image on the left).
+여기서의 출력은 형태(21, H, W)이며, 각 위치에는 각 클래스의 예측에 해당하는 정규화되지 않은 확률이 있습니다. 각 클래스의 최대 예측을 가져온 다음 이를 다운스트림 작업에 사용하려면 'output_propertions = output.slmax(0)'를 수행합니다. 다음은 각 클래스에 할당된 각 색상과 함께 예측을 표시하는 작은 토막글 입니다(왼쪽의 시각화 이미지 참조).
```
Comment (JDhyeok) [inline pytorch_vision_fcn_resnet101.md:65]: `'output_propertions = output.slmax(0)'` 부분 코드블럭을 사용하실 의도라면 작은 따옴표 대신 ` 을 사용하시면 될 것 같아요.

```suggestion
여기서의 출력은 형태(21, H, W)이며, 각 위치에는 각 클래스의 예측에 해당하는 정규화되지 않은 확률이 있습니다. 각 클래스의 최대 예측을 가져온 다음 이를 다운스트림 작업에 사용하려면 `output_propertions = output.slmax(0)`를 수행합니다. 다음은 각 클래스에 할당된 각 색상과 함께 예측을 표시하는 작은 토막글 입니다(왼쪽의 시각화 이미지 참조).
```

사소한 오타수정 요청드립니다. :)

**Review by JDhyeok — CHANGES_REQUESTED** [review]

**Comment by Taeyoung96** [comment]
@cepiloth   
@JDhyeok 님이 남겨주신 review에 대해 update 부탁드립니다.  😄

File: pytorch_vision_fcn_resnet101.md (hunk 6 above)
Comment (cepiloth) [reply pytorch_vision_fcn_resnet101.md]: @JDhyeok 감사합니다. 해당내역 반영 하도록 하겠습니다.

**Review by cepiloth — COMMENTED** [review]
팀원들이 제시한 내용 및 첨언을 통하여 번역을 수정 하였습니다.

**Comment by Taeyoung96** [comment]
@cepiloth   
아직 Review 업데이트가 잘 반영이 안된 것 같습니다.  
제가 언급한 `데이터 세트`도 `데이터셋`으로 변경이 안된 것 같은데, 한번 확인해주시면 감사드리겠습니다. :smile:

**Review by Taeyoung96 — APPROVED** [review]
피드벡이 잘 반영된 것 같습니다.  
고생하셨습니다. 😄

**Review by codingbowoo — APPROVED** [review]
이해하기에 어려운 부분 없이 잘 번역된 것으로 보입니다.
수고하셨습니다!
